### PR TITLE
Change custom blocks to contain any node type

### PR DESCRIPTION
**Pull request by ohakutsu**
### Change

Allow a custom block to contain any node type.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 
+- [#19](https://github.com/increments/qiita_marker/pull/19): Allow all nodes to included in custom blocks.
```
Comment (kyntk) [inline CHANGELOG.md:9]: Don't you upgrade the version?

File: ext/qiita_marker/qfm_mention_no_emphasis.c [hunk 2]
```diff
@@ -1,5 +1,5 @@
-#include "cmark_ctype.h"
 #include "cmark-gfm.h"
+#include "cmark_ctype.h"
```
Comment (kyntk) [inline ext/qiita_marker/qfm_mention_no_emphasis.c:2]: :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (ohakutsu) [reply CHANGELOG.md:9]: @kyntk 
How about releasing it later, as we plan to make some other changes?

**Review by atm-snag2 — APPROVED** [review]
LGTM

**Review by kyntk — APPROVED** [review]
LGTM!

**Pull request merged**